apps/sertra/README.md

```diff
@@ -0,0 +1,34 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
+
+First, run the development server:
+
+```bash
+npm run dev
+# or
+yarn dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
```

Comment:
@thunderstore/nextjs also runs on port 3000, should this use 3001 oslt by default?

Edit: Oh, it seems the dev server will automatically move to 3001 if 3000 is in use. Anyways, could still avoid some confusion?

Reply:
Perhaps, but the other side of the coin is that 3000 is the default port and if you're only running a single project, you'd probably expect it to go to port 3000. So I'd keep it as is given that the project is developed standalone for the time being.